packages/angular/projects/ui-angular/src/lib/services/authenticator.service.ts

```diff
@@ -22,29 +29,25 @@ const logger = new Logger('state-machine');
 })
 export class AuthenticatorService implements OnDestroy {
   private _authState: AuthMachineState;
+  private _authStatus: AuthStatus = 'configuring';
   private _authService: AuthInterpreter;
   private _machineSubscription: Subscription;
   private _facade: ReturnType<typeof getServiceFacade>;
+  private _hubSubject: Subject<void>;
```

Comment:
[`Subject`](https://rxjs.dev/guide/subject) is a special type of Observable. You can push new values to it, and you can also subscribe to it.

We use it here so that `AuthenticatorService` can push new values when hub events occur, and `Authenticator` can subscribe it to respond to them.